packages/webapp-libs/webapp-crud-demo/src/routes/crudDemoItem/crudDemoItemDetails/crudDemoItemDetails.component.tsx

```diff
@@ -22,12 +27,32 @@ export const CrudDemoItemDetails = () => {
   };
   const { id } = useParams<keyof Params>() as Params;
   const { data: currentTenant } = useCurrentTenant();
+  const { toast } = useToast();
+  const intl = useIntl();
+  const navigate = useNavigate();
+  const generateLocalePath = useGenerateLocalePath();
+  const errorMessage = intl.formatMessage({
+    id: 'CrudDemoItem / CrudDemoItemDetails / ErrorMessage',
+    defaultMessage: 'Failed to load item details',
+  });
+  const notFoundMessage = intl.formatMessage({
+    id: 'CrudDemoItem / CrudDemoItemDetails / NotFoundMessage',
+    defaultMessage: 'The requested item does not exist',
+  });
 
   const { loading, data } = useQuery(crudDemoItemDetailsQuery, {
     variables: {
       id,
       tenantId: currentTenant?.id || '',
     },
+    onError: (error) => {
+      if (error.message.includes('No CrudDemoItem matches the given query')) {
```

Comment:
This is not a good idea to branch message on the message from the backend. Better would be to use a error code if possible. Basically this message can be easily changed because it's localized.